packages/cli/src/cmds/agentInstaller/install-agent.ts

```diff
@@ -85,6 +86,26 @@ class InstallerError {
 
       UI.error(`${chalk.red('!')} ${this.error.message}`);
       return true;
+    } else if (this.error instanceof UserConfigError) {
+      UI.error(
+        `${chalk.red('!')} Error in project configuration:\n\n${
+          this.error?.message
+        }`
+      );
+
+      Telemetry.sendEvent({
+        name: 'install-agent:user_config_error',
```

Comment:
We seem to have several different conventions for naming telemetry event, but snake-case is currently the least-used. I'd vote for making this either `user-config-error` or `userConfigError`, with a preference for the former.